money-akka/src/main/scala/com/comcast/money/akka/stream/TracedStreamCombinators.scala

```diff
@@ -0,0 +1,171 @@
+package com.comcast.money.akka.stream
+
+import akka.actor.ActorSystem
+import akka.stream._
+import akka.stream.scaladsl.GraphDSL.Builder
+import akka.stream.scaladsl.{Concat, Flow, GraphDSL, Partition, Source, Unzip, Zip}
+import com.comcast.money.akka.{MoneyExtension, SpanContextWithStack}
+import com.comcast.money.core.Tracer
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect.ClassTag
+
+trait AkkaMoney {
+  implicit val actorSystem: ActorSystem
+
+  lazy val moneyExtension: MoneyExtension = MoneyExtension(actorSystem)
+
+  def tracer(spanContext: SpanContextWithStack,
+             moneyExtension: MoneyExtension): Tracer =
+    moneyExtension.tracer(spanContext)
+}
+
+trait TracedStreamCombinators {
+  self: AkkaMoney =>
+
+  import DefaultSpanKeyCreators._
+  import akka.stream.scaladsl.GraphDSL.Implicits._
+
+  implicit class PortOpsSpanInserter[In: ClassTag](portOps: PortOps[(In, SpanContextWithStack)])
+                                                  (implicit builder: Builder[_],
+                                                   fskc: FlowSpanKeyCreator[In] = DefaultFlowSpanKeyCreator[In]) {
+    type TracedIn = (In, SpanContextWithStack)
+
+    def ~|>[Out: ClassTag](flow: Flow[In, Out, _]): PortOps[(Out, SpanContextWithStack)] =
```

Comment:
unclear if these make sense when called, but in general when using symbols for operators, I recommend that they are aliases for another function with a more descriptive name.

Also, really need scaladoc on all of these things

Like if `~|>` means `injectFlow` there should be a function called `injectFlow`

Reply:
I guess on the making sense part it's primarily directed at people who already use GraphDSL but yeah I totally get what you mean.

Sure I absolutely can do on converting in to an underlying named function.

hundred percent on the scala doc will do now.